test_no_talisman.py

```
@@ -0,0 +1,63 @@
+    with app.app_context():
+        db.create_all()
+
+    return app
+
+if __name__ == '__main__':
+    app = create_test_app()
+
+    # Force HTTP without SSL for testing
+    port = int(os.getenv('CULTIVAR_PORT', 5000))
+    print("Starting test app on HTTP (no HTTPS redirect)...")
+    app.run(host='0.0.0.0', port=port, debug=True, use_reloader=False)
```

Ruff (0.13.1)

Possible binding to all interfaces

(S104)